### PR TITLE
Introduce pkg/service test suite

### PR DESCRIPTION
**Pull request by rgolangh**
- Don't log the whole request because it may contain secrets
- Remove obsoleted node_test.go
- Fix lint errors
- Add pkg/service test suite with NodeService tests
- Add IdentityService tests

CNV-8315

### CODE REVIEW

**Comment by rgolangh** [comment]
@ydayagi please take a look

File: pkg/service/node.go [hunk 1]
```diff
@@ -21,23 +21,60 @@ import (
...
+			return os.MkdirAll(path, perm)
+		},
+	}
 }
 
 var nodeCaps = []csi.NodeServiceCapability_RPC_Type{
 	csi.NodeServiceCapability_RPC_STAGE_UNSTAGE_VOLUME,
 }
 
+type callableFunc func() ([]byte, error)
```
Comment (rgolangh) [inline pkg/service/node.go:48]: I'm going to simplify this a bit, and maybe make the function type interfaces type

File: pkg/service/service_suite_test.go [hunk 2]
```diff
@@ -1,13 +1,294 @@
...
 )
 
+const serialID = "4b13cebc-7406-4c19-8832-7fcb1d4ac8c5"
+
 func TestService(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "Service Suite")
 }
+
+var _ = Describe("NodeService", func() {
```
Comment (ydayagi) [inline pkg/service/service_suite_test.go:25]: I ,personally , prefer the Go testing package. I find it simpler and more readable.

**Review by ydayagi — APPROVED** [review]

**Pull request merged**